packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -16,12 +17,19 @@ function InputDateTimePicker(props) {
 			'Warning: "selectedDateTime" is deprecated and will be removed in the next major version. Use "value" instead please.',
 		);
 	}
+	if (props.useUTC) {
+		// eslint-disable-next-line no-console
+		console.warn(
+			'Warning: "useUTC" is deprecated and will be removed in the next major version. Set "timezone" instead please.',
+		);
```

Comment:
I didn't know that we have to deprecate the utc mode. Is it a request from UX ?

Reply:
No, it's not request from UX, just I supposed it. because `useUTC` is equivalent with `timezone='UTC'`.

Reply:
Oh ok I didn't know that there was an `UTC` value in timezone.
Then, we can deprecate it, but in the pickers, if we have the useUTC props, just set the timezone to `UTC`, and then you can remove all the `if(useUTC)` code.
With that we still manage the useUTC props, adding the warning to pass `timezone="UTC"` instead, but only have the timezone code.

Reply:
@jsomsanith Oh sorry, actually `Etc/UTC` should be the timezone value to use. Then should we deprecate it? (also take @rmannibucau 's comment into account: https://github.com/Talend/ui/pull/2422#issuecomment-539973061)

Reply:
I will merge this one to feature branch for now so I can continue work. I will author another PR to address useUTC and timezone issue.